Add CUAV V5+ Build Log

```diff
@@ -0,0 +1,176 @@
+* Battery: [3S 5200mAh]()
+
+
+## Hardware
+
+Description | Quantity
+---|---
+
+Unibody Frame Plate           | 1
+Flight Controller Cover Plate | 1
+Camera Plate                  | 1
+35mm Standoffs                | 6
+Vinyl screws & Nuts           | 4
+15mm Steel Screws             | 8
+Steel Nuts                    | 8
+7mm Steel Screws              | 12
+Velcro Battery Strap          | 1
+Foam for Battery              | 1
+Landing Pads                  | 4
+
+## Package
+Description | Quantity (Default Package) | Quantity (GPS Package)
+---|---|---
+
+V5+ Autopilot              |      1     |     1  
+DuPont Cable               |      2     |     2
+I2C/CAN Cable              |      2     |     2
+ADC 6.6 Vable              |      2     |     2
+SBUS Signal Cable          |      1     |     1
+IRSSI Cable                |      1     |     1
+DSM Signal Cable           |      1     |     1
+ADC 3.3 Cable              |      1     |     1
+Debug Cable                |      1     |     1
+Safety Switch Cable        |      1     |     1
+Voltage & Current Cable    |      1     |     1
+PW-Link Module Cable       |      1     |     1
+Power Module               |      1     |     1
+SanDisk 16GB Memory Card   |      1     |     1
+12C Expansion Boward       |      1     |     1
+TTL Plate                  |      1     |     1
+NEO GPS                    |      -     |     1
+GPS Bracket                |      -     |     1
+
+### Electronics
+Description | Quantity
+--- | --- 
+
+Motors - DR2205 KV2300                            |     4
+Fully assembled Power Management Board with ESCs  |     4
+Holybro power module                              |     1
+Fr-sky D4R-II receiver                            |     1
+CUAV V5+                                  |     1
+Holybro GPS Neo-M8N                               |     1
+Battery lumenier 1300 mAh 4S 14.8V               |     1
+Vtx 5.8gHz                                        |     1
+FPV camera                                        |     1
+
+### Tools needed
+
+The following tools are used in this assembly:
+
+- 2.0mm Hex screwdriver
+- 3mm Phillips screwdriver
+- Wire cutters
+- Precision tweezers
+
+## Assembly
+
+Estimated time to assemble frame is 2 hours and 1.5 hours installing the autopilot and configuring the airframe in *QGroundControl*.
+
+Step 1: Attach the 4 armed to the bottom plate as seen in figure 1.
+
+Step 2:  Solder ESC (Electronic Speed Controller) to the board, Red to the positive side and Black for the negative side as shown on figure 2. 
+
+Step 3: Solder the Power Module, positive (red) and negative (black).
+See figure 3.
+
+Step 4: Plug in the motors to the ESCs according to its positions. For example on Figure 4.   
+
+Step 5: Attach the motors to the corresponding arm, see example in figures 5 and 6
+
+Step 6: Place the top board as shown on figure 7
+
+Step 7: Paste the damping foam on the CUAV V5+ flight controller.
```

Comment:
Is it internally damped? If so, should it not be double sided tape?

Reply:
It is double sided taped.